README.md

```diff
@@ -430,6 +430,8 @@ Table of Contents
   * [GraphCMS](https://graphcms.com/) - Offers free tier for small projects. GraphQL first API. Move away from legacy solutions to the GraphQL native Headless CMS - and deliver omnichannel content API first.
   * [Squidex](https://squidex.io/) - Offers free tier for small projects. API / GraphQL first. Open source, and based on event sourcing (versing every changes automatically).
   * [InstaWP](https://instawp.com/) - Launch a WordPress site in a few seconds. A free tier with: 5 Active Sites, 500 MB Space, 48 hrs Site Expiry.
+  * [Strapi](https://strapi.io/) -  Headless CMS open-source, Node.js based. Use to connect the front-end with the Strapi back-end.  
```

Comment:
Is the backend delivered as a SaaS and does it have a free tier?

Reply:
Currently its only available for self hosting, the cloud version is coming soon, so please add it only after the cloud version is available with a free tier

Reply:
Cool. Will close this then you can open a new one when ready. Thank you